assets/dev/scss/editor/panel/controls/_globals.scss

```diff
@@ -137,12 +137,32 @@
 		}
 
 		&-preview {
-			height: 20px;
-			width: 20px;
-			border-radius: 3px;
-			border: 1px solid rgba(0, 0, 0, 0.1);
-			@include margin-end(10px);
-			flex-shrink: 0;
+
+			&-container {
+				height: 20px;
+				width: 20px;
+				border-radius: 3px;
+				border: 1px solid rgba(0, 0, 0, 0.1);
+				@include margin-end(10px);
+				flex-shrink: 0;
+				position: relative;
+			}
+
+			&-color,
+			&-transparent-bg {
+				border-radius: 3px;
+				height: 100%;
+				width: 100%;
+				position: absolute;
+				top: 0;
+				left: 0;
+			}
+
+			&-transparent-bg {
+				background-image: linear-gradient(45deg,#ddd 25%,transparent 0,transparent 75%,#ddd 0,#ddd),linear-gradient(45deg,#ddd 25%,transparent 0,transparent 75%,#ddd 0,#ddd);
+				background-size: 12px 12px;
+				background-position: 0 0,6.5px 6.5px;
```

Comment:
Export it to a mixin that can be used everywhere with one parameter of a checker size